explorer/Explorer.scss

```diff
@@ -116,10 +116,6 @@ html.IsInIframe .ExplorerFigure,
         position: relative;
         min-height: 600px;
 
-        // The chart normally leaves 5px padding on the sides for the shadows. In the future we may
-        // want to approach this better, but this is a quick and dirty fix for now.
-        margin: -2.5px;
-
         // Remove chart from layout calculation.
```

Comment:
Removing this made the alignment slightly worse on explorers with controls (see [sandbox](https://86jv8h.csb.app/), explorer with controls, live vs local)